src/main/xar-resources/data/security/security.xml

```diff
@@ -190,120 +271,100 @@
         </listitem>
       </itemizedlist>
 
-      <para>Given the previous example, we can see that all files except
-          <literal>r_and_j.xml</literal> are owned by user <code>admin</code> and group
-          <code>dba</code>.</para>
-      <para>As mentioned in the previous section, the database root collection
-          <literal>/db</literal> initially has its permissions set to <literal>drwxr-xr-x</literal>,
-        so full access is granted to everyone. </para>
+
+      <para>The database root collection <literal>/db</literal> initially has its permissions set to
+          <literal>drwxr-xr-x</literal>, so full access is granted to everyone. You might consider
+        changing this.</para>
       <para> Also note that the default setting for all newly created resources is
-          <literal>-rw-r--r--</literal>: the owner has read/write permissions, not execute, and the
+          <literal>-rw-r--r--</literal>: the owner has read/write permissions, not execute. The
         group and others (world) have read permissions only.</para>
 
       <sect3>
-        <title>Changing Resource Permissions</title>
-
-        <para>Permissions can be changed using either the <guimenuitem>Edit Properties</guimenuitem>
-          dialog box (shown below) in the <link xlink:href="java-admin-client">Java Admin
-            Client</link> or the <literal>chmod</literal> command in its shell window. </para>
-        <para>The <guimenuitem>Edit Properties</guimenuitem> dialog box is opened by selecting
-            <guimenuitem>Files, Resource Properties</guimenuitem> from the menu, or by clicking on
-          the <guimenuitem>Properties Icon</guimenuitem> (image with checkboxes) in the toolbar.
-          This dialog box shows the permission settings for all database users and groups.</para>
-        <informalfigure>
-          <mediaobject>
-            <imageobject>
-              <imagedata fileref="assets/client-6.png" width="50%"/>
-            </imageobject>
-          </mediaobject>
-        </informalfigure>
+        <title>Changing Resource Permissions and Ownership</title>
+
         <note>
           <para>Please note that only the owner of a resource or members of the
               <literal>dba</literal> group are allowed to change permissions. All other users who
             attempt to change these settings will receive an error message.</para>
         </note>
-        <para>On the command line, you can use the <literal>chmod</literal> command. This command
-          expects two parameters:</para>
-        <programlisting>chmod [resource] [user|group|other]=[+|-][read|write|execute][, ...]</programlisting>
-        <orderedlist>
+
+        <para>To change permissions and/or ownership, use one of the following methods: </para>
+        <itemizedlist>
           <listitem>
-            <para>Name of a resource or collection</para>
+            <para>Use the GUI of the .</para>
```

Comment:
~~of the .~~

Reply:
Done. BTW: The security article is not complete yet. I identified  gaps and asked Adam directly to supply me with information so i can finish it.

Reply:
👍